src/pages/transactions/[id].js

```diff
@@ -41,6 +54,14 @@ export default function TransactionDetails() {
         <h2>Description</h2>
         <div>{transaction.description}</div>
       </div>
+      {mode === "delete" && (
+        <Modal
+          message="Are you sure you want to delete this entry?"
+          hint="This will delete this entry permanently and cannot be undone."
+          onConfirm={() => onDelete(id)}
```

Comment:
Another point I noticed when testing in the preview: While styling is currently not a priority, I find the complete lack of styling is now beginning to interfere with the usability of the app. Thus, both the modal and the alert that pops up after successful deletion are hard to differentiate from the rest of the page content, since they blend in perfectly. You may want to add a simple border around these components to make them stand out a bit.

Reply:
modal and alert have been visually separated from the rest of the content and can now also be found at the top of the list